components/Op/OpDetailForm.js

```diff
@@ -371,18 +371,31 @@ class OpDetailForm extends Component {
             </DescriptionContainer>
             <InputContainer>
               <ShortInputContainer>
-                <Form.Item label={opCommitment}>
+                <Form.Item
+                  label={opCommitment}
+                  name='Commitment'
+                >
                   {getFieldDecorator('duration', {
                     rules: [
                       {
-                        required: false,
+                        required: this.state.requiredForPublish,
                         message: 'Commitment level is required'
                       }
                     ]
-                  })(<Input placeholder='4 hours' />)}
+                  })(<Input className='commitment' placeholder='4 hours' />)}
                 </Form.Item>
-                <Form.Item label={opStartDate}>
-                  {getFieldDecorator('startDate', {})(
+                <Form.Item
+                  label={opStartDate}
+                  name='Start date'
+                >
+                  {getFieldDecorator('startDate', {
+                    rules: [
+                      {
+                        required: this.state.requiredForPublish,
+                        message: 'Start date is required'
```

Comment:
Date is not a required field.
activities can be open and take place in the background e.g. negotiable. 
online activities may be done whenever the volunteer wants to do it.

Reply:
Dates were listed as a mandatory requirement in the card....??

Reply:
yeah sorry about that - the card was wrong.